numpy/char.py

```diff
@@ -0,0 +1,2 @@
+import numpy as np
+from numpy.core.defchararray import *
```

Comment:
(https://github.com/numpy/numpy/pull/16370/files#r494150059 applies here too)

Reply:
The addition of this file means that `from numpy.char import count` is legal, when previously it wasn't. I don't feel too strongly about it, but that perhaps deserves a release note and/or a separate PR.